api/app/GraphQL/Mutations/DuplicatePool.php

```diff
@@ -3,6 +3,7 @@
 namespace App\GraphQL\Mutations;
 
 use App\Models\Pool;
+use Database\Helpers\KeyStringHelpers;
```

Comment:
This seems like an accidental addition?

Reply:
Leftover from an intermediate commit https://github.com/GCTC-NTGC/gc-digital-talent/pull/7087/commits/5479a16ee5c539c4058d7fa05a5f9fe215ebd2a1